backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -324,22 +328,7 @@ export class TransactionResolver {
 
     // validate recipient user
     const recipientUser = await findUserByEmail(email)
-    /*
-    const emailContact = await UserContact.findOne({ email }, { withDeleted: true })
-    if (!emailContact) {
-      logger.error(`Could not find UserContact with email: ${email}`)
-      throw new Error(`Could not find UserContact with email: ${email}`)
-    }
-    */
-    // const recipientUser = await dbUser.findOne({ id: emailContact.userId })
-
-    /* Code inside this if statement is unreachable (useless by so), 
-    in findUserByEmail() an error is already thrown if the user is not found
-    */
-    if (!recipientUser) {
-      logger.error(`unknown recipient to UserContact: email=${email}`)
-      throw new Error('unknown recipient')
-    }
```

Comment:
?

Reply:
Code is never reached. `findUserByEmail` throws if user cannot be found